lib/dateTime.js

```diff
@@ -1167,3 +1167,40 @@ export function formatTimeFromTimestamp(timestamp, options) {
 	const date = parseLocalDateTimeFromTimestamp(timestamp);
 	return formatTime(date, options);
 }
+
+export function formatRelativeDateTime(date, { numeric = 'always', style = 'long' } = {}) {
+
+	if (!Intl.RelativeTimeFormat) {
+		const now = new Date();
+		const d = new Date(date);
+		if (d.getDate() === now.getDate()) {
+			return formatTime(d);
+		}
+		return formatDate(d);
+	}
+
+	const rtf = new Intl.RelativeTimeFormat(getLanguage(), {
```

Comment:
The `localeMatcher: 'best fit'` option does quite a good job of resolving an appropriate code. Now, for `fr-on` we probably would prefer `fr-ca` and not `fr`, but that's going to be an issue no matter what because of how wonky D2L codes are. We're might be better off with a map 😢.

<img width="545" alt="Screen Shot 2023-01-10 at 4 17 13 PM" src="https://user-images.githubusercontent.com/1289042/211665059-a68d6b4e-7b78-485e-aff1-4dab9ace2d83.png">

Reply:
Right. I'm good with how it is and we can adjust if we run into issues.